src/Native/WebGL.js

```diff
@@ -52,66 +52,72 @@ var _elm_community$webgl$Native_WebGL = function () {
   }
 
   function doSettings(gl, settings) {
+    var disable = [];
```

Comment:
I can call it cleanup, its a list of features to roll back, so the next render will start with the clean environment.